docs/content/manual/pre-release/managed-kubernetes-clusters/AKS/expand-volume.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "[Expand Volume](https://longhorn.io/docs/1.3.0/advanced-resources/support-managed-k8s-service/manage-node-group-on-aks/)"
+---
+1. Create AKS cluster with 3 nodes and install Longhorn.
+2. Create [deployment](https://github.com/longhorn/longhorn/blob/master/examples/deployment.yaml) and write some data to it.
+3. In Longhorn, set `replica-replenishment-wait-interval` to `0`.
+4. Add a new node-pool. Later Longhorn components will be automatically deployed on the nodes in this pool.
+
+    ```
+    AKS_NODEPOOL_NAME_NEW=<new-nodepool-name>
+    AKS_RESOURCE_GROUP=<aks-resource-group>
+    AKS_CLUSTER_NAME=<aks-cluster-name>
+    AKS_DISK_SIZE_NEW=<new-disk-size-in-gb>
+    AKS_NODE_NUM=<number-of-nodes>
+    AKS_K8S_VERSION=<kubernetes-version>
+    az aks nodepool add \
+      --resource-group ${AKS_RESOURCE_GROUP} \
+      --cluster-name ${AKS_CLUSTER_NAME} \
+      --name ${AKS_NODEPOOL_NAME_NEW} \
+      --node-count ${AKS_NODE_NUM} \
+      --node-osdisk-size ${AKS_DISK_SIZE_NEW} \
+      --kubernetes-version ${AKS_K8S_VERSION} \
+      --mode System
+    ```
+5. Using Longhorn UI to disable the disk scheduling and request eviction for nodes in the old node-pool.
+6. Cordon and drain Kubernetes nodes in the old node-pool.
+    ```
+    AKS_NODEPOOL_NAME_OLD=<old-nodepool-name>
+    for n in `kubectl get nodes | grep ${AKS_NODEPOOL_NAME_OLD}- | awk '{print $1}'`; do
+      kubectl cordon $n && \
+      kubectl drain $n --ignore-daemonsets --delete-emptydir-data
```

Comment:
For draining the node, suggest excluding CSI attacher/provisioner pods to ensure volume operations correctly between longhorn CSI node plugin and kubelet. The same change to other managed K8s distros. 

ref: https://longhorn.io/docs/1.2.4/volumes-and-nodes/maintenance/#updating-the-node-os-or-container-runtime